notebooks/samples/explanations/tf2/ai-explanations-image.ipynb

```diff
@@ -1,1771 +1,1636 @@
 {
```

Comment:
<p>You can use the <a href="https://pypi.org/project/explainable-ai-sdk/" target="_blank">Explainable AI SDK</a> to generate most of the fields <s>for us. </s></p><p> </p><p>Can you add a line of code that prints out the metadata before exporting it?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/200/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>

Reply:
Done.